android/app/src/bt/java/covidsafepaths/bt/exposurenotifications/common/NotificationHelper.java

```diff
@@ -0,0 +1,72 @@
+package covidsafepaths.bt.exposurenotifications.common;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationCompat.Builder;
+import androidx.core.app.NotificationManagerCompat;
+
+import org.pathcheck.covidsafepaths.MainActivity;
+import org.pathcheck.covidsafepaths.R;
+
+import java.util.Objects;
+
+import static org.pathcheck.covidsafepaths.MainActivity.ACTION_LAUNCH_FROM_EXPOSURE_NOTIFICATION;
+
+/**
+ * Helper class for managing notifications in the app.
+ */
+public final class NotificationHelper {
+
+    private static final String EXPOSURE_NOTIFICATION_CHANNEL_ID = "EXPOSURE_NOTIFICATION_CHANNEL_ID";
+
+    /**
+     * Shows a notification, notifying of a possible exposure.
+     */
+    public static void showPossibleExposureNotification(Context context) {
+        createNotificationChannel(context);
+        Intent intent = new Intent(context, MainActivity.class);
```

Comment:
We are just opening `MainActivity` once the user taps on the notification
Should we do something else?

Reply:
This is good for now!